pcsx2/gui/MainFrame.cpp

```diff
@@ -429,7 +429,7 @@ MainEmuFrame::MainEmuFrame(wxWindow* parent, const wxString& title)
 	m_menuSys.AppendSeparator();
 
 	m_menuSys.Append(MenuId_EnablePatches,	_("Automatic &Gamefixes"),
-		_("Automatically applies needed Gamefixes to known problematic games"), wxITEM_CHECK);
+		_("Automatically applies needed Gamefixes."), wxITEM_CHECK);
```

Comment:
So, I just remembered that you can't see some of the help texts in the main frame. Is it helpful to make this string shorter if you can't see it anyway in the main window?

Reply:
Furthermore, "Automatically applies needed Gamefixes." doesn't tell the whole truth.
If you incist on making it shorter, "Automatically applies Gamefixes." would be better.

Reply:
At the very least it doesn't clip anymore even if the text is not visible from the menu, so it's still a very slight improvement.

Reply:
Before -  Can't see anything.
After - Can't see anything.

Not sure you can call it an improvement.

Reply:
I can purge it if you're fine with that.

Reply:
Well, problem is I don't know if they're useful or not.

Reply:
I'll just close this then, nothing noteworthy to change.